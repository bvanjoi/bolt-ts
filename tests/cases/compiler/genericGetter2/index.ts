// From `github.com/microsoft/TypeScript/blob/v5.8.2/tests/cases/compiler/genericGetter2.ts`, Apache-2.0 License

class A<T> { }

class C<T> {
    data: A<T>;
    get x(): A { //~ ERROR: Generic type 'A<T>' requires 1 type argument.
        return this.data;
    }
}