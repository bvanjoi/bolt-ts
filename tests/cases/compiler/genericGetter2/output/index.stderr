  × Generic type 'A<T>' requires 1 type argument.
   ╭─[../../tests/cases/compiler/genericGetter2/index.ts:7:14]
 7 │ A
   · ─
   ╰────
